src/ucp/proto/proto_multi.inl

```diff
@@ -271,6 +271,30 @@ ucp_proto_multi_lane_map_progress(ucp_request_t *req, ucp_lane_map_t *lane_map,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_proto_eager_bcopy_multi_start_send(
```

Comment:
do not change the function in proto_multi.inl
instead create new function in proto_am.inl that also handles user header copy